Automattic/wp-calypso
# Allow setting anonymous tracks user ID when redirecting into authorize flow

Builds on changes in w.js in D7080-code which allow setting the anonId to a new value without losing aliasing of prior events.

To test: 
- Apply D7080-code to your WPCOM sandbox
- Go through Jetpack Connect auth flow and log in
- You should observe calls to pixel.wp.com/t.gif which alias anon user `jetpack:XXXXXX` to the WPCOM user, AND the existing, Calypso-generated anonId
- The `ai` cookie should be set to the `jetpack:XXXXXX` value
